runtime/runtime-params-estimator/src/cases.rs

```diff
@@ -609,8 +609,9 @@ fn get_ext_costs_config(measurement: &Measurements) -> ExtCostsConfig {
         promise_and_per_promise: measured_to_gas(metric, &measured, promise_and_per_promise),
         promise_return: measured_to_gas(metric, &measured, promise_return),
         // TODO: accurately price host functions that expose validator information.
-        validator_stake_base: measured_to_gas(metric, &measured, validator_stake_base),
-        validator_total_stake_base: measured_to_gas(metric, &measured, validator_total_stake_base),
+        validator_stake_base: measured_to_gas(metric, &measured, storage_write_value_byte) * 1024,
```

Comment:
This seems like a major change. What is the reason to increase the cost of these operations?
Also, I think we should simply price it high, but not attach it to any other specific fee, e.g. we could price it at the cost of 1/10 of the max gas per contract execution.

Reply:
> This seems like a major change. What is the reason to increase the cost of these operations?

This code doesn't work before because we don't have proper measurement of the operation.

> Also, I think we should simply price it high, but not attach it to any other specific fee, e.g. we could price it at the cost of 1/10 of the max gas per contract execution.

That doesn't work given how the voting contract works. We potentially need to invoke it 100 times in one call.